lib/active_model/serializer.rb

```diff
@@ -27,32 +29,62 @@ class Serializer
       )
     /x
 
-    class << self
-      attr_accessor :_attributes
-      attr_accessor :_attributes_keys
+    # Hashes contents of file for +_cache_digest+
```

Comment:
Could you elaborate (just for my personal understanding) on which practical situations the error is thrown?